### ISSUE TITLE
Training is unable to produce effective models, and the output appears to be gibberish

### ISSUE THREAD

**Issue by JayZhao315** [clovaai/donut#213]
Hello,

I am very interested in this project and believe it has a wide range of applications. However, as someone with very little programming experience, I've spent several days trying to get it working, to no avail. I haven't made any adjustments to the code, I've only changed the configuration.

For this project, I'm using the Chinese business card dataset mentioned in the project and have selected 1000 cards from it as a training set. Here are my training configurations:

> resume_from_checkpoint_path: null
> result_path: "/root/autodl-tmp/donut/result"
> pretrained_model_name_or_path: "/root/autodl-tmp/donut-base"
> dataset_name_or_paths: ["/root/autodl-tmp/data_train/mp_1000_v1"] 
> sort_json_key: True
> train_batch_sizes: [8]
> val_batch_sizes: [1]
> input_size: [320, 640]
> max_length: 512
> align_long_axis: False
> num_nodes: 1
> seed: 2022
> lr: 3e-5
> warmup_steps: 1200
> num_training_samples_per_epoch: 850
> max_epochs: 100
> max_steps: -1
> num_workers: 8
> val_check_interval: 1.0
> check_val_every_n_epoch: 10
> gradient_clip_val: 1.0
> verbose: True

Here are the issues I've encountered:

1. During training, the feedback from the validation set is as shown in the image below. Despite the epoch just starting in the graph, there's no difference even after training up to 100 rounds.
![image](https://github.com/clovaai/donut/assets/128065547/34b30b82-8fd6-40e5-aff1-21d0c8232803)
<img width="897" alt="image" src="https://github.com/clovaai/donut/assets/128065547/8706327b-dadc-43ab-b66d-d3ba0262c263">

2. The model trained shows a score of 0 in both categories when testing.
3. During model loading, I see a prompt stating 
`Some weights of the model checkpoint at /root/autodl-tmp/donut-base were not used when initializing DonutModel: ['encoder.encoder.layers.`
`This IS expected if you are initializing DonutModel from the checkpoint of a model trained on another task or with another architecture (e.g. initializing a BertForSequenceClassification model from a BertForPreTraining model).`
`This IS NOT expected if you are initializing DonutModel from the checkpoint of a model that you expect to be exactly identical (initializing a BertForSequenceClassification model from a BertForSequenceClassification model).`I'm unsure whether this is something I should pay attention to.

Please note: There might be network connection issues with my training server to huggingface, so I have preemptively downloaded the `asian-bart-ecjk` and `swin_base_patch4_window12_384` models and placed them in the `.cache` folder under `huggingface` and `torch`.

I would greatly appreciate any advice or suggestions on how to resolve these issues. Thank you in advance! 

Best regards,

**Comment by Mohtadrao** [comment]
How did the solved it? As I am having a similar error.
@JayZhao315 
![image](https://github.com/clovaai/donut/assets/58603622/546f4488-3877-4a78-9a01-666485ce1357)

**Comment by JayZhao315** [comment]
> 它是如何解决的？因为我有类似的错误。 ![image](https://private-user-images.githubusercontent.com/58603622/297727418-546f4488-3877-4a78-9a01-666485ce1357.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTcxNDUyNzQsIm5iZiI6MTcxNzE0NDk3NCwicGF0aCI6Ii81ODYwMzYyMi8yOTc3Mjc0MTgtNTQ2ZjQ0ODgtMzg3Ny00YTc4LTlhMDEtNjY2NDg1Y2UxMzU3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA1MzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNTMxVDA4NDI1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwODk0OTJhMWU3YjcyZjQ3ZmQwN2IwZjQyNzI3NDRiODBiZDEyNmRlYTI2M2FhNWJmZWQ2NDY3MDgzMDE2MGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.CrZUYJxyB9-V1YZxzy6MoV6GVfyPbImSm1ZcoP8B8oQ)

项目中依赖其他的两个模型,需要联网下载
